autoibamr.sh

```diff
@@ -177,8 +178,7 @@ while [ -n "$1" ]; do
             echo "  --enable-native-optimizations          Build dependencies and IBAMR with platform-specific optimizations."
             echo "                                         This option is compatible with --enable-assertions-with-optimizations."
             echo "  --enable-numdiff                       Build the numdiff tool, which is required for IBAMR's test suite."
-            echo "                                         Numdiff depends on gettext (available through homebrew) on macOS and has"
-            echo "                                         no external dependencies on Linux."
```

Comment:
This was fixed by #159 - I verified that we only link against a system library on macs.